lib/HttpClient.ts

```diff
@@ -202,6 +206,16 @@ export class HttpClient {
         return response;
     }
 
+    public destroy() {
```

Comment:
Let's call this dispose.  Put a comment on the method that it only needs to be called if keepAlive is true.  This should also set a disposed flag and other operations should fail if disposed.

Reply:
done.